### PR TITLE
[Migration] [Translation i18(fr)] Migrate debian and adding the french version for v2

### LINKED ISSUE #2190: [docs] Migrate  Distribute Debian Package
### Question you want answered

When I want to package my app as a .deb I want to know any related configuration of it 

### Where did you look for an answer?

I am running one of the app since v2 was in alpha and in v2 the `tauri.conf.json` has changed now  the v1 is wrong at some place so after looking a bit in it I was able to figured out how it work. So I decided to start making the migration for it. 
Also I made the french version for it. 

### Page URL

https://v2.tauri.app/distribute/debian/

### Additional context

If there is other config  in for the deb please tell me I will add it . Also if could give more  info for the rpm and appimage  config would be appreciated because it will be the next thing i will work on.

### Are you willing to work on this yourself?

- [X] I want to work on this myself

### PR DESCRIPTION
**Pull request by Sir-Thom**
#### What kind of changes does this PR include?
- Migrate the doc for the configuration of the .deb package 
- Added the french version of the doc for the configuration of the .deb package 
- 
#### Description
This PR migrate the the configuration of the .deb package  and Add french translation for it 
- Closes #2190

### CODE REVIEW

**Comment by vasfvitor** [comment]
hey, thanks for this and welcome :). Somebody (or me) will review this later, now I'm just wondering why was tauri submodule deleted?

**Comment by Sir-Thom** [comment]
> hey, thanks for this and welcome :). Somebody (or me) will review this later, now I'm just wondering why was tauri submodule deleted?

Oops that  a  dumb mistake on my part it should be readded now

**Comment by vasfvitor** [comment]
ok, no problem

File: src/content/docs/_fr/distribute/debian.mdx
Comment (dreyfus92) [inline src/content/docs/_fr/distribute/debian.mdx]:
```suggestion
La version de base du paquet Debian généré par le bundler de Tauri, contient tout ce dont vous avez besoin pour distribuer votre application sur les distributions Linux basées sur Debian. Il définit les icônes de votre application, génère un fichier Desktop et spécifie les dépendances `libwebkit2gtk-4.0-37` et `libgtk-3-0`, avec `libappindicator3-1` si votre application utilise la zone de notification du système.
```

File: src/content/docs/distribute/debian.mdx
Comment (dreyfus92) [inline src/content/docs/distribute/debian.mdx]:
```suggestion
If your app depends on additional system dependencies you can specify them in `tauri.conf.json > bundle > linux > deb`.
```

File: src/content/docs/_fr/distribute/debian.mdx
Comment (dreyfus92) [inline src/content/docs/_fr/distribute/debian.mdx]:
```suggestion
Si votre application dépend de dépendances système supplémentaires, vous pouvez les spécifier dans `tauri.conf.json > bundle > linux > deb`.
```

**Review by dreyfus92 — COMMENTED** [review]
Salut @Sir-Thom, voici quelques suggestions 😁

**Review by dreyfus92 — APPROVED** [review]
Merci @Sir-Thom, LGTM ✅

File: pnpm-lock.yaml
Comment (simonhyll) [inline pnpm-lock.yaml:1]: There's no need to commit an updated lockfile for translations

File: src/content/docs/_fr/distribute/Pipelines/github.mdx
Comment (simonhyll) [inline src/content/docs/_fr/distribute/Pipelines/github.mdx:1]: Stub files should not be committed to translations, it breaks the intended setup where you get shown the fallback english page

File: src/content/docs/_fr/distribute/Pipelines/gitlab.mdx
Comment (simonhyll) [inline src/content/docs/_fr/distribute/Pipelines/gitlab.mdx:1]: No stub pages for translations

File: src/content/docs/_fr/distribute/Pipelines/index.mdx
Comment (simonhyll) [inline src/content/docs/_fr/distribute/Pipelines/index.mdx:1]: No stub pages for translations

File: src/content/docs/_fr/distribute/app-store.mdx
Comment (simonhyll) [inline src/content/docs/_fr/distribute/app-store.mdx:1]: No stub pages for translations

File: src/content/docs/_fr/distribute/debian.mdx
Comment (simonhyll) [inline src/content/docs/_fr/distribute/debian.mdx]: Unnecessary import

File: src/content/docs/_fr/distribute/index.mdx
Comment (simonhyll) [inline src/content/docs/_fr/distribute/index.mdx:1]: No stub pages for translations

File: src/content/docs/_fr/distribute/play-store.mdx
Comment (simonhyll) [inline src/content/docs/_fr/distribute/play-store.mdx:1]: No stub pages for translations

File: src/content/docs/_fr/distribute/steam.mdx
Comment (simonhyll) [inline src/content/docs/_fr/distribute/steam.mdx:1]: No stub pages for translations

File: src/content/docs/distribute/debian.mdx
Comment (simonhyll) [inline src/content/docs/distribute/debian.mdx]: Unnecessary import now that it's been written

**Review by simonhyll — CHANGES_REQUESTED** [review]
Nice work, just needs a bit of fixing :)

File: src/content/docs/distribute/debian.mdx
Comment (tweidinger) [inline src/content/docs/distribute/debian.mdx]: I think these are v1 dependencies and it won't work in v2

File: src/content/docs/distribute/debian.mdx
Comment (tweidinger) [inline src/content/docs/distribute/debian.mdx]: I think the v1 example was simpler to understand with example files (https://v1.tauri.app/v1/guides/building/linux#custom-files):
```
  "files": {
          "/usr/share/README.md": "../README.md", // copies the README.md file to /usr/share/README.md
          "usr/share/assets": "../assets/" // copies the entire assets directory to /usr/share/assets
        }
```

**Review by tweidinger — CHANGES_REQUESTED** [review]
Can you please double check the dependencies, as from my understanding it should be webkit2gtk `4.1` instead of `4.0` and not sure about the other libraries.

I would remove the stub files in `_fr/distribute` (no translation -> no file at all) as there is no content added and I think the file names already changed with recent PRs.

File: src/content/docs/distribute/debian.mdx
Comment (tweidinger) [inline src/content/docs/distribute/debian.mdx]:
```suggestion
          "/usr/share/assets": "../assets/" // copies the entire assets directory to /usr/share/assets
```
Very small nitpick that is also wrong in v1 docs.

File: src/content/docs/fr/distribute/debian.mdx
Comment (tweidinger) [inline src/content/docs/fr/distribute/debian.mdx]:
```suggestion
          "/usr/share/assets": "../assets/" // copies toutes le dossier assets dans /usr/share/assets
```
Very small nitpick that is also wrong in v1 docs.

**Review by tweidinger — CHANGES_REQUESTED** [review]
I think this is good to go aside from the very small nitpick on the example file, which I will also port to v1 (#2638 )🙃 . Will approve once last suggestions are merged.

Thanks for the porting!

**Review by tweidinger — APPROVED** [review]

**Comment by tweidinger** [comment]
There is some prettyfier formatting check failing. Will fix once back at the laptop in some hours (otherwise feel free to fix yourself). Else good to go 👍

**Pull request merged**
